### ISSUE TITLE
VSIX fails to install on 17.10 preview. Missing CodingConventions prerequisite

### ISSUE THREAD

**Issue by pharring** [pharring/EditorGuidelines#122]
An internal Microsoft user reports that the EditorGuidelines VSIX fails to install on an internal preview (17.10.18-preview1+xxx)
The error message says:

```
Microsoft.VisualStudio.ExtensionManager.MissingReferencesException: This extension cannot be installed because the following references are missing:
- Microsoft.VisualStudio.CodingConventions (Microsoft Visual Studio CodingConventions)
```

The error message is self-explanatory. I need to find out why CodingConventions is missing from that installation.

**Comment by pharring** [comment]
This needs more work. The extension installs OK now, but Guidelines no longer work.

**Comment by pharring** [comment]
Fixed in 2.1.11
